example/synchronous_resnet/resnet_controller.py

```diff
@@ -0,0 +1,150 @@
+from __future__ import absolute_import, print_function
+import os
+import sys
+import time
+
+import numpy
+
+sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..'))
+from platoon.channel import Controller
+
+
+class ResNetController(Controller):
```

Comment:
For now, just reuse the LSTMController directly. We can see what we do later for a general controler